packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -193,28 +233,24 @@ Future<Map<String, String>?> getCodeSigningIdentityDevelopmentTeam({
     return null;
   }
 
-  final String? developmentTeam = _certificateOrganizationalUnitExtractionPattern
-      .firstMatch(opensslOutput)
-      ?.group(1);
-  if (developmentTeam == null) {
-    return null;
-  }
-
-  return <String, String>{
-    'DEVELOPMENT_TEAM': developmentTeam,
-  };
+  return _certificateOrganizationalUnitExtractionPattern.firstMatch(opensslOutput)?.group(1);
 }
 
 Future<String?> _chooseSigningIdentity(
   List<String> validCodeSigningIdentities,
   Logger logger,
   Config config,
   Terminal terminal,
+  bool shouldExitOnNoCerts,
 ) async {
   // The user has no valid code signing identities.
   if (validCodeSigningIdentities.isEmpty) {
-    logger.printError(noCertificatesInstruction, emphasis: true);
-    throwToolExit('No development certificates available to code sign app for device deployment');
+    if (shouldExitOnNoCerts) {
```

Comment:
This is a little clunky, but the old code was called when a code signing cert was actually necessary on build, and the new code wants to non-fatally check if there's a saved cert, and if there isn't just skip using any in the create template.

Reply:
can you add a comment giving a hint about the `shouldExitOnNoCerts == false` behavior?